query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ServicesQueries.java

```diff
@@ -25,12 +29,8 @@
    */
   static QueryRequest buildQuery1() {
     Builder builder = QueryRequest.newBuilder();
-    ColumnIdentifier serviceId = ColumnIdentifier.newBuilder().setColumnName("SERVICE.id").build();
-    ColumnIdentifier serviceName =
-        ColumnIdentifier.newBuilder()
-            .setColumnName("SERVICE.name")
-            .setAlias("SERVICE.name")
-            .build();
+    ColumnIdentifier serviceId = createColumnIdentifier("SERVICE.id");
```

Comment:
nit: we introduced a new util function to build an identifier, but from what I can see we just turn around and wrap it in an expression - and we already have a util to create a column expression.